migrating_from_ocp_3_to_4/installing-3-4.adoc

```diff
@@ -0,0 +1,63 @@
+[id='installing-3-4']
+= Installing the Migration Toolkit for Containers
+include::modules/common-attributes.adoc[]
+:context: installing-3-4
+
+toc::[]
+
+You can install the {mtc-full} ({mtc-short}) on {product-title} 3 and on {product-title} {product-version} clusters.
+
+[IMPORTANT]
+====
+You must install the same {mtc-short} version on all clusters.
+====
+
+By default, the {mtc-short} web console and the `Migration Controller` pod run on the target cluster.
+
+You can configure the `Migration Controller` custom resource manifest to run the {mtc-short} web console and the `Migration Controller` pod on a link:https://access.redhat.com/articles/5064151[source cluster or on a remote cluster].
+
+== Installing the {mtc-short} Operator in a restricted network environment
+
+If you are installing the {mtc-short} Operator in a restricted network environment, you must create a mirror image of the Operator catalog, push it to a local registry, and then create a catalog from the image.
+
+[discrete]
+=== Additional resources
```

Comment:
I don't know what this looked like before, but this "Additional resources" section in the middle threw me off. Is this list of resources specific to the above restricted network environment "module"? If so, then I think that people have been using the ".Additional resources" heading so that it looks like it's paired with the above section.